views/search/search.html

```diff
@@ -9,4 +9,5 @@ <h4 class="jw-content-title">
 
   <jw-card-grid feed="vm.feed" on-card-click="vm.cardClickHandler" card-class-name="jw-lazy-load"
       cols="{xs:2, sm:2, md:3, lg:4, xl:5}"></jw-card-grid>
-</div>
\ No newline at end of file
+  <jw-button ng-if="vm.itemsLeft()" ng-click="vm.showMoreClickHandler()" class="jw-button-default jw-button-secondary jw-button-show-more jw-card-grid">Show more results...</jw-button>
```

Comment:
Break up attributes over newlines

Reply:
Either we do that with all the html elements in this file or we don't

Reply:
@RWOverdijk I suggest to do this all over the file

Reply:
Only for long lines. Then, yes. I agree.